### PR TITLE
Add sourceinfo mixin to segment definition for 2p audience share

### PR DESCRIPTION
**Pull request by knarayanan88**
Please link to the issue #…
Issue => #1150

### CODE REVIEW

File: extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/segmentdefinition-sourceinfo",
```
Comment (mjasrotia) [inline extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json:8]: Please change it to
"$id": "https://ns.adobe.com/xdm/mixins/segmentdefinition-sourceinfo"

File: extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
```
Comment (mjasrotia) [inline extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json:1]: Please place the file under the following directory:
xdm/components/mixins/segment-definition

File: extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  ],
+  "description": "Segment Definition Source/Ownership Mixin with details such as Source Customer, FeedName etc. in context of 2P Audience Sharing AEP initiative",
+  "definitions": {
+    "segmentdefinition-sourceinfo": {
+      "xdm:segmentSourceInfo": {
+        "properties": {
+          "xdm:segmentOwner": {
+            "title": "Segment Owner",
+            "type": "string",
+            "description": "The customer org name that shared this segment"
```
Comment (mjasrotia) [inline extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json:25]: Description should end with a period "."

File: extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+          },
+          "xdm:feedName": {
+            "title": "Feed Name",
+            "type": "string",
+            "description": "Name of the feed of which, the segment is a part of"
```
Comment (mjasrotia) [inline extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json:30]: Description should end with a period "."

**Comment by prabhum2** [comment]
@knarayanan88 - Please let us know if the recommended changes are done and ready to review by XDM team?

File: extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json (hunk 1 above)
Comment (cmathis) [reply extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json:8]: Actually, this is not a standard XDM mixin so it should not be in the mixin namespace. The $id should be something like "https://ns.adobe.com/adobe/experience/audience-sharing/segmentdefinition-sourceinfo"

Comment (cmathis) [reply extensions/adobe/experience/audience-sharing/segmentdefinition-sourceinfo.schema.json:8]: Nevermind, what you have here is correct

**Comment by knarayanan88** [comment]
Closing this now  Product is now looking at reusing existing SegmentFoldering mixin to achieve what I am trying to do with this mixin.

**Pull request closed without merging**